app/package.json

```diff
@@ -4,6 +4,8 @@
   "description": "Modelling groups burden contribution portal",
   "main": "index.js",
   "scripts": {
+    "webpack": "webpack",
+    "watch_webpack": "webpack --watch",
```

Comment:
`npm run webpack_watch` is actually more commands to type than `webpack --watch`, or am I missing something?